### DISCUSSION TITLE
Showcase

### DISCUSSION THREAD

**Discussion by zenorocha** [dracula/dracula-ui#61] [Show and tell]
Share what you are building with Dracula UI!

I'll start...

- Title: Dracula UI Homepage
- Link: https://draculatheme.com/ui
- Source code: https://github.com/dracula/draculatheme.com/blob/site/pages/ui.js

<img width="1307" alt="Screen Shot 2021-04-14 at 5 28 14 AM" src="https://user-images.githubusercontent.com/398893/114710148-3cb57900-9ce2-11eb-9734-24f431277c20.png">

**Comment by Rixcy** [comment]
Title: rickbooth.me (personal website)
Link: https://rickbooth.me
Source code: currently private but might make it public shortly just need to clean up a bit 😅 

![image](https://user-images.githubusercontent.com/11819124/114830924-ad34c680-9dc4-11eb-93ce-d8b009aafa7c.png)
![image](https://user-images.githubusercontent.com/11819124/114831142-f2f18f00-9dc4-11eb-8af5-a7ad87765d7b.png)

**Reply by a-metta** [reply]
Looking fresh!

**Reply by zenorocha** [reply]
I love it!

**Reply by mgoggin** [reply]
This looks great! Clean and simple.

**Comment by pbteja1998** [comment]
- Title: My Personal Website (WIP)
- Link: https://bhanuteja.dev

<img width="1552" alt="preview" src="https://user-images.githubusercontent.com/17903466/115293695-42dda680-a175-11eb-9dc6-b2686aaa8b79.png">

<img width="1552" alt="Screenshot 2021-04-20 at 1 12 05 AM" src="https://user-images.githubusercontent.com/17903466/115293928-8e905000-a175-11eb-8841-4e32207152ab.png">

<img width="1552" alt="Screenshot 2021-04-20 at 1 12 27 AM" src="https://user-images.githubusercontent.com/17903466/115293961-9ea82f80-a175-11eb-93d8-7d7e4fc9a45a.png">

<img width="1552" alt="Screenshot 2021-04-20 at 1 15 27 AM" src="https://user-images.githubusercontent.com/17903466/115294482-47ef2580-a176-11eb-99a5-6172d8bfd477.png">

**Reply by nettofarah** [reply]
Love the contribution chart. This is beautiful 🤩

**Reply by zenorocha** [reply]
This looks amazing!

**Comment by vinicoder** [comment]
Title: DracBoard (Dashboard)
Link: https://dracula-dashboard.vercel.app/
Source code: I'll make it public when I'm done 🤓

![Screen Shot 2021-04-20 at 11 31 29](https://user-images.githubusercontent.com/3522573/115414038-17d27180-a1cc-11eb-84e3-1b70becf6992.png)

**Reply by brunogenaro** [reply]
@vinicoder you are the man! 👏🏻👏🏻

**Reply by nettofarah** [reply]
Sweeeeet!

**Reply by flosrn** [reply]
Really great! 🥇

**Reply by neelhabib** [reply]
did you make it public?

**Comment by brunogenaro** [comment]
Title: WebSolutionsFL (Company Website)
Link: https://www.websolutionsfl.com/
Source code: private

Props to @vinicoder for building the website! I'm really happy with the final result.

<img width="1904" alt="Screen Shot 2021-06-11 at 8 00 25 AM" src="https://user-images.githubusercontent.com/3578942/121688252-3bf65480-ca91-11eb-8399-3db6515ca527.png">

<img width="1904" alt="Screen Shot 2021-06-11 at 8 00 45 AM" src="https://user-images.githubusercontent.com/3578942/121688255-3c8eeb00-ca91-11eb-975a-a6f981b84de1.png">

**Reply by flosrn** [reply]
Really great work! So beautiful website 😍

**Comment by didair** [comment]
Title: Personal website (aekstrom.me)
Link: https://aekstrom.me/
Source code: https://github.com/didair/aekstrom.me

![bild](https://user-images.githubusercontent.com/2635185/121791647-7fb1a100-cbec-11eb-9ffe-7ba5024e1a8c.png)

**Comment by rohitkumarankam** [comment]
Title: My Personal Website
Link: [https://rohitkumarankam.com](https://rohitkumarankam.com)

![image](https://user-images.githubusercontent.com/70012972/171051014-5ed56180-4964-403d-8ef2-7d6f733b7862.png)

**Comment by aorumbayev** [comment]
Title: AlgoWorld Swapper
Link: https://swapper.algoworld.io/
Repo: https://github.com/AlgoWorldNFT/algoworld-swapper

<img width="1440" alt="Screenshot 2022-06-05 at 12 28 21" src="https://user-images.githubusercontent.com/7698600/172046197-f9846925-1fb5-4d28-9171-bccc1b0dfae7.png">

Not using the UI directly but color pallete is heavily inspired by dracula ;)

**Comment by TCNOco** [comment]
Title: TroubleChute Tech-Help Hub
Link: https://hub.tcno.co/
Source code: private

Very loosely based around Dracula, and Dracula Van Helsing/Cyan.

| Homepage  | Article  | List page |
|---|---|---|
| ![hub tcno co_](https://user-images.githubusercontent.com/10319195/188463746-b391f5ce-2778-450e-b762-5a3b94269f29.png)  |   ![screenshot-hub tcno co-2022 09 05-15_43_40](https://user-images.githubusercontent.com/10319195/188463749-84447d3d-c448-4a8b-9e7d-62c933eb5d8d.png) |  ![screenshot-hub tcno co-2022 09 05-15_47_39](https://user-images.githubusercontent.com/10319195/188464293-0201ec77-0483-4502-ae14-1e93fab66c4d.png) |

No idea how to better show images so I've just done them in a table :)
